pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -278,6 +287,15 @@ protected void consumeLoop() {
         Uninterruptibles.sleepUninterruptibly(100, TimeUnit.MILLISECONDS);
       }
     }
+
+    _serverMetrics.addMeteredTableValue(_metricKeyName, ServerMeter.ROWS_WITH_ERRORS,
```

Comment:
These metrics are also updated when segment is created. Do we want to them updated everytime we call consumeLoop()? (Can be for catching up as well).

Also, will there be double-counting? (See logStatisics() method)